rules/vhost/vhost.go

```diff
@@ -0,0 +1,50 @@
+package vhost
+
+import (
+	"gopkg.in/vinxi/vinxi.v0/config"
+	"gopkg.in/vinxi/vinxi.v0/mux"
+	"gopkg.in/vinxi/vinxi.v0/rule"
+)
+
+const (
+	// Name exposes the rule name identifier.
+	Name = "vhost"
+	// Description exposes the rule semantic description.
+	Description = "Matches HTTP request by Host header"
+)
+
+// params defines the rule specific configuration params.
+var params = rule.Params{
+	rule.Field{
+		Name:        "host",
+		Type:        "string",
+		Description: "Hostname expression to match. Regular expressions are supported.",
+		Mandatory:   true,
+	},
+}
+
+// Rule exposes the rule metadata information.
+// Mostly used internally.
+var Rule = rule.Info{
```

Comment:
Not sure it needs to be exported. @h2non maybe you can give me your opinion on this. Is this more for convinience or is there no real reason for it to be exported?